msal/src/main/res/values/public.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like it can be removed

Reply:
This actually has purpose.  By declaring something public... you make everything not declared public private.  With that said... I have not been able to verify the documented behavior yet... since i've been loading the library locally and not as an package.